hazelcast/src/main/java/com/hazelcast/instance/Node.java

```diff
@@ -666,10 +669,13 @@ public SplitBrainJoinMessage createSplitBrainJoinMessage() {
     public JoinRequest createJoinRequest(boolean withCredentials) {
         final Credentials credentials = (withCredentials && securityContext != null)
                 ? securityContext.getCredentialsFactory().newCredentials() : null;
+        final InternalHotRestartService hotRestartService = nodeExtension.getInternalHotRestartService();
+        final Set<String> excludedMemberUuids = hotRestartService != null
```

Comment:
Instead of `null` checking everywhere, can we have a nop implementation of `InternalHotRestartService` in OSS? 
So `nodeExtension.getInternalHotRestartService()` never returns `null`...